CHANGELOG.md

```diff
@@ -6,9 +6,24 @@ Mapbox welcomes participation and contributions from everyone.
 
 #### Features
 
+#### Bug fixes and improvements
+
+## Mapbox Navigation SDK 2.3.0-rc.2 - February 23, 2022
+### Changelog
+[Changes between v2.3.0-rc.1 and v2.3.0-rc.2](https://github.com/mapbox/mapbox-navigation-android/compare/v2.3.0-rc.1...v2.3.0-rc.2)
+
 #### Bug fixes and improvements
 - Added a workaround for [`ConnectivityManager`'s occasional security exception on Android 11 and older](https://issuetracker.google.com/issues/175055271). [#5492](https://github.com/mapbox/mapbox-navigation-android/pull/5492)
 
+### Mapbox dependencies
+This release depends on, and has been tested with, the following Mapbox dependencies:
+- Mapbox Maps SDK `v10.3.0` ([release notes](https://github.com/mapbox/mapbox-maps-android/releases/tag/android-v10.3.0))
+- Mapbox Navigation Native `v88.0.0`
+- Mapbox Core Common `v21.1.1`
+- Mapbox Java `v6.3.0-beta.1` ([release notes](https://github.com/mapbox/mapbox-java/releases/tag/v6.3.0-beta.1))
```

Comment:
```suggestion
- Mapbox Java `v6.3.0` ([release notes](https://github.com/mapbox/mapbox-java/releases/tag/v6.3.0))
```

Reply:
Was typing the same https://github.com/mapbox/mapbox-navigation-android/pull/5499 😅

Reply:
Thanks I was looking at the wrong screen.